engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -136,6 +138,23 @@ object MaterializeWorkflowDescriptorActor {
       CallCachingOff.validNel
     }
   }
+
+  // perform a fail-fast validation that the `memory_retry_multiplier` workflow option is valid if present
+  def validateMemoryRetryMultiplier(workflowOptions: WorkflowOptions): ErrorOr[Unit] = {
+    val optionName = WorkflowOptions.MemoryRetryMultiplier.name
+    workflowOptions.get(optionName) match {
+      case Success(value) => refineV[MemoryRetryMultiplier](value.toDouble) match {
+        case Left(_) =>
+          s"Workflow option '$optionName' is invalid. It should be in the range 1.0 <= n <= 99.0".invalidNel
+        case Right(_) =>
+          ().validNel
+      }
+      case Failure(OptionNotFoundException(_)) =>
```

Comment:
You should be able to do a `get[Option[String]]` to avoid this awkward case

Reply:
For some reason I am not able to find `workflowOptions.get[Option[String]](optionName)` here. All the `.get` methods on workflow options return a `Try[String]`. Maybe I am missing something?

Reply:
I think I might be thinking of reading values from config. Looks fine to me.